Automattic/pocket-casts-android
# Migrate player heading section to Compose

## Description

This migrates player heading section with chapter prev, next buttons to Compose.

## Testing Instructions
1. Play a podcast episode
2. Open full screen player
3. ✅ Notice that episode and podcast titles are rendered properly 
4. Tap on podcast title
5. ✅ Notice that podcast is opened
6. Play an episode with chapters
7.  ✅ Notice that chapter title, summary, and prev, next chapter buttons  are rendered properly 
8. Tap on chapter title
9. ✅ Notice that Chapters tab is opened with the chapter in progress selected
10. Go back to Now Playing tab
11. Tap on prev/ next chapter buttons
12. ✅ Notice that playback moves to prev/ next chapter position
13. ✅ Notice that chapter progress is reflected properly in the chapter progress circle surrounding the next chapter button

## Screenshots or Screencast 

Style Previews

With Chapter
<img width="525" src="https://github.com/user-attachments/assets/7db5fa9d-114e-485e-b437-dcd21ec24ff8"/>


Without Chapter
<img width="525" src="https://github.com/user-attachments/assets/9cbb5cb3-d0c6-4008-8d12-c39b3dfd48f5">


Portrait

Without Chapter | With Chapter
----|----
<img width=200 src="https://github.com/user-attachments/assets/bdedd8c3-dda5-4f67-956f-1be22a1bf5d0"/>| <img width=200 src="https://github.com/user-attachments/assets/06c4c3b6-71e0-4288-b38c-4369dd60cc48"/>

Landscape Without Chapter
<img width=500 src="https://github.com/user-attachments/assets/9d66117f-33f7-4459-a3d5-f737173341bd"/>


Landscape With Chapter
<img width=500 src="https://github.com/user-attachments/assets/4be381ae-fb1d-44b4-b21d-3f10c1fab923"/>

Large Display

<img width=500 src="https://github.com/user-attachments/assets/7dfb8f02-5a64-4454-8a00-613b42bf7c2f"/>


Tablet 
<img width=500 src="https://github.com/user-attachments/assets/d755bcfc-6bad-4a6f-b9b5-b27443a9b3ee"/>



## Checklist
 
#### I have tested any UI changes...
